internal/plugins/manifests/v2/init.go

```diff
@@ -155,14 +155,26 @@ IMAGE_TAG_BASE ?= %[1]s/%[2]s
 # BUNDLE_IMG defines the image:tag used for the bundle.
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+
+# USE_IMAGE_DIGESTS defines if images are resolved via tags or digests
+# You can enable this value if you would like to use SHA Based Digests
+# To enable set flag to true
+USE_IMAGE_DIGESTS ?= false
+
+# BUNDLE_GEN_FLAGS is an additional mechanism for resolving running sdk generate bundle flags
+BUNDLE_GEN_FLAGS ?= ""
+
+ifeq ($(USE_IMAGE_DIGESTS) true)
+	BUNDLE_GEN_FLAGS += "--use-image-digests"
+endif
```

Comment:
```suggestion

# BUNDLE_GEN_FLAGS are the flags passed to the operator-sdk generate bundle command
BUNDLE_GEN_FLAGS ?= "-q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)"

# USE_IMAGE_DIGESTS defines if images are resolved via tags or digests
# You can enable this value if you would like to use SHA Based Digests
USE_IMAGE_DIGESTS ?= false
ifeq ($(USE_IMAGE_DIGESTS), true)
	BUNDLE_GEN_FLAGS += " --use-image-digests"
endif
```

Has this been tested? What you originally had was missing a comma in the ifeq call. Regen with my changes then make sure `make bundle` works in the test project.

Reply:
```
memcached-operator/bin/kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version 0.0.1   --use-image-digests
```

^^ above tested output with sample memcached operator